int32 senderID
string masterPlan
string currentPlan
string currentState
string currentRole
string currentTask
int32[] robotIDsWithMe
